applications/GeoMechanicsApplication/tests/cpp_tests/test_reset_displacement_process.cpp

```diff
@@ -0,0 +1,117 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "custom_processes/reset_displacement_process.h"
+#include "geo_mechanics_fast_suite.h"
+#include <boost/numeric/ublas/assignment.hpp>
+#include <includes/deprecated_variables.h>
+
+namespace
+{
+
+using namespace Kratos;
+
+class StubConstitutiveLaw : public ConstitutiveLaw
+{
+};
+
+class StubElement : public Element
+{
+public:
+    KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(StubElement);
+
+    StubElement(IndexType NewId, GeometryType::Pointer pGeometry) : Element(NewId, pGeometry)
+    {
+        for (int i = 0; i < 3; ++i) {
+            mConstitutiveLaws.push_back(std::make_shared<StubConstitutiveLaw>());
+        }
+    }
+
+    void CalculateOnIntegrationPoints(const Variable<ConstitutiveLaw::Pointer>& rVariable,
+                                      std::vector<ConstitutiveLaw::Pointer>&    rOutput,
+                                      const ProcessInfo& rCurrentProcessInfo) override
+    {
+        rOutput = mConstitutiveLaws;
+    }
+
+    void CalculateOnIntegrationPoints(const Variable<Vector>& rVariable,
+                                      std::vector<Vector>&    rOutput,
+                                      const ProcessInfo&      rCurrentProcessInfo) override
+    {
+        rOutput = mIntegrationVectorValues;
+    }
+
+    using Element::CalculateOnIntegrationPoints;
+
+    std::vector<Vector>                   mIntegrationVectorValues = {};
+    std::vector<ConstitutiveLaw::Pointer> mConstitutiveLaws        = {};
+};
+
+ModelPart& CreateModelPartWithAStubElement(Model& model)
```

Comment:
Just to follow the Kratos Style Guide:
```suggestion
ModelPart& CreateModelPartWithAStubElement(Model& rModel)
```

Reply:
Done